lib/components/settings/settings.dart

```diff
@@ -84,16 +86,16 @@ class _SettingsHomeState extends State<_SettingsHome> {
                       bool isSelected = provider._pageIndex == index;
 
```

Comment:
avoid nested ternary operators, only use max one of these per element, having more makes the code messy to read and understand